src/OtpInput/OtpInput.tsx

```diff
@@ -90,7 +90,8 @@ export const OtpInput = forwardRef<OtpInputRef, OtpInputProps>((props, ref) => {
         autoFocus={autoFocus}
         style={styles.hiddenInput}
         secureTextEntry={secureTextEntry}
-        autoComplete="one-time-code"
+        // @ts-ignore
+        autoComplete={Platform.OS === 'ios' ? "one-time-code" : "sms-otp"}
```

Comment:
as "sms-otp" is specific to Android but "one-time-code" is more general prop, it would be better to change the condition in other way around, like this:

``
autoComplete={Platform.OS === 'android' ? "sms-otp" : "one-time-code"}
``